### PR TITLE
Updating my caption

### PR DESCRIPTION
**Pull request by Chocrates**
Updating my slide's caption

### CODE REVIEW

File: _slides/02-Chocrates.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello World!
```
Comment (gitM-iru) [inline _slides/02-Chocrates.md:6]: Looks good

**Review by gitM-iru — COMMENTED** [review]
Please add comment.

**Review by tommeenabong — CHANGES_REQUESTED** [review]
Add hello to line 5

**Review by tommeenabong — APPROVED** [review]
awesome!

**Review by SonLe20663 — APPROVED** [review]

File: _slides/02-Chocrates.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ layout: slide
...
+Hi it's Chris!
 
 ![femalecodertocat](https://octodex.github.com/images/femalecodertocat.png)
```
Comment (Chocrates) [inline _slides/02-Chocrates.md:9]: test

**Pull request merged**